### PR TITLE
Pythia8 interfaces for decay

### PR DESCRIPTION
**Pull request by jncots**
@HDembinski, I need a couple interfaces for using Pythia8 decay capabilities. I use them for some time privately. I guess it will be useful to have those functions exposed anyway when the decay capabilities of Pythia8 will be introduced in chromo. Can I add them now?

### CODE REVIEW

File: src/cpp/_pythia8.cpp [hunk 1]
```diff
@@ -209,6 +209,7 @@ PYBIND11_MODULE(_pythia8, m)
         .def("init", &Pythia::init, py::call_guard<py::scoped_ostream_redirect, py::scoped_estream_redirect>())
         .def("next", py::overload_cast<>(&Pythia::next))
         .def("readString", &Pythia::readString, "setting"_a, "warn"_a = true)
+        .def("forceHadronLevel", &Pythia::forceHadronLevel, "find_junctions"_a = true)
```
Comment (HDembinski) [inline src/cpp/_pythia8.cpp:212]: I cannot check right now (on my phone): forceHadronLevel should support the same arguments as the C++ version. If that's the case, it is all good.

File: src/cpp/_pythia8.cpp [hunk 2]
```diff
@@ -244,6 +245,8 @@ PYBIND11_MODULE(_pythia8, m)
         .def("vt", event_array_v<Vec4_tt>)
         .def("parents", event_array_parents)
         .def("children", event_array_children)
+        .def("reset", &Event::reset)
+        .def("append", py::overload_cast<int, int, int, int, double, double, double, double, double, double, double>(&Event::append), "pdgid"_a, "status"_a, "col"_a, "acol"_a, "px"_a, "py"_a, "pz"_a, "e"_a, "m"_a = 0, "scale"_a = 0, "pol"_a = 9.)
```
Comment (HDembinski) [inline src/cpp/_pythia8.cpp:249]: Same point as above.

**Review by HDembinski — COMMENTED** [review]
Look good to me, see minor comments.

**Review by HDembinski — APPROVED** [review]

File: src/cpp/_pythia8.cpp (hunk 1 above)
Comment (jncots) [reply src/cpp/_pythia8.cpp:212]: I checked it once again. forceHadronLevel has only one argument. I use a default value.  It seems to work fine.

File: src/cpp/_pythia8.cpp (hunk 2 above)
Comment (jncots) [reply src/cpp/_pythia8.cpp:249]: I use only one of the versions of append and arguments to that overload are the same as in this interface.

**Pull request merged**